src/NuGet.Core/NuGet.Build.Tasks/NuGet.Build.Tasks.csproj

```diff
@@ -40,6 +46,10 @@
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildPackageVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildPackageVersion)" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.9.0" />
```

Comment:
Where is this used?

Reply:
It's used by the logging queue: https://github.com/NuGet/NuGet.Client/pull/3109/files#diff-4b17c63a81b794616ee9ad377f6d5c80R5